srcpkgs/jrnl/template

```diff
@@ -13,6 +13,6 @@ license="GPL-3.0-only"
 homepage="https://jrnl.sh"
 changelog="https://raw.githubusercontent.com/jrnl-org/jrnl/develop/CHANGELOG.md"
 distfiles="${PYPI_SITE}/j/jrnl/jrnl-${version}.tar.gz"
-checksum=089c7f2e61c31809b96efc4e3d3ae9fcc6354257e27406754f3a4a1a37029574
+checksum=3838f0b55708b9b9b2c2706fb671bd83f444d321635e3ce19cda2304992d4f60
 # Tarball provides no tests
 make_check=no
```

Comment:
if we switch distfiles to github, we will get a tarball with tests

Reply:
doesn't build with setuptools